dotcom-rendering/src/client/bootCmp.ts

```diff
@@ -91,7 +96,13 @@ export const bootCmp = async (): Promise<void> => {
 			document.readyState === 'complete'
 		) {
 			// Manually updates the footer DOM because it's not hydrated
-			injectPrivacySettingsLink();
+			void schedule(
+				'inject privacy settings link in footer',
+				() => Promise.resolve(injectPrivacySettingsLink()),
+				{
+					priority: 'low',
+				},
+			);
```

Comment:
Would this be “low” priority? I do not think we are allowed to forgo the privacy settings link on a page, according to the law.

Would there be a setting that ensures something is run eventually? Maybe akin to [“idle until urgent”](https://philipwalton.com/articles/idle-until-urgent/)?

Reply:
we discussed this irl, but just to capture that here:

it's a very good point. if it's possible we'll end up not runing something, that should be clearer. 

maybe we remove this functionality?

or as an alternative, priorities of `'critical' | 'optional' | undefined` where:

`critical` = _must_ try to run it
`optional` = may try to run it after everything else
`undefined` = will try to run it after all critical things have run

Reply:
made this change, so resolving this (for now...possibly!)